source/user-manual/reference/ossec-conf/vuln-detector.rst

```diff
@@ -76,14 +76,17 @@ Runs updates and vulnerabilities scans immediately when service is started.
 ignore_time
 ^^^^^^^^^^^^
 
-Time during which vulnerabilities that have already been alerted will be ignored.
+Time during which vulnerabilities that have already been alerted will be ignored. When this time hasn't passed yet, only partial scans will be performed.
```

Comment:
Probably we could add a linked reference to `partial scans`.

Reply:
The `Scan types` section was referenced.